CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
+## 2.1.0 (April 2, 2021)
+
```

Comment:
It looks good that there is no blank line after the version.

Reply:
okay.